src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -43,6 +43,6 @@
        "dnxcore50": { }
     },
     "runtimes": {
-       "win7-x86": { }
+       "win7-x64": { }
```

Comment:
Why?

If we do add this, do we need to do something so we depend on the 32 bit versions of Microsoft.NETCore packages?